lib/src/rules/throw_in_finally.dart

```diff
@@ -49,13 +49,20 @@ class BadThrow {
 ''';
 
 class ThrowInFinally extends LintRule {
+  static const LintCode code = LintCode(
+      'throw_in_finally', "Use of '{0}' in finally block.",
+      correctionMessage: "Try moving the '{0}' outside the finally block.");
```

Comment:
Above, in `control_flow_in_finally`, "finally" is quoted. e.g. "outside the 'finally' block."

Reply:
Done. Sometimes I'm not as consistent as I should be, so thanks for pointing this out.